.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
@EricFromCanada When splitting this out into a new PR: could you make a separate one just for the non-Cask-specific changes like this (and the formula changes) so I can merge that immediately? Thanks!

Reply:
Submitted as #121.